attainCors.ts

```diff
@@ -0,0 +1,52 @@
+import { CorsOptions, CorsOptionsDelegate } from "./types.ts";
+import { Cors } from "./cors.ts";
+import { Request } from "https://deno.land/x/attain/mod.ts";
+import { CallBackType } from "https://deno.land/x/attain/types.ts";
```

Comment:
@aaronwlee I think since your _Request_ and _CallBackType_ probably have different definition in different attain versions(we have exact same issue with Oak #1) we should provide some sort of independent definitions that just satisfies our requirements([Req](https://github.com/tajpouria/cors/blob/master/oakCors.ts#L4) [Res](https://github.com/tajpouria/cors/blob/master/oakCors.ts#L11) [MiddleWareT](https://github.com/tajpouria/cors/blob/master/oakCors.ts#L27))), What are think?

Reply:
@tajpouria I understood what you mean.
As I followed the requirements, I added independent Request, Response, and middleware definitions. Thanks :)